src/event.rs

```diff
@@ -0,0 +1,241 @@
+use accesskit::Vec2;
+
+/// Kittest Event.
+pub enum Event {
+    /// An ActionRequest event. When using an application these would be generated by the
+    /// screen reader.
+    ActionRequest(accesskit::ActionRequest),
+    /// A Simulated event, e.g. clicks or typing.
```

Comment:
nit: there is some weird casing going on here

```suggestion
/// Kittest event sent to the GUI application.
pub enum Event {
    /// An [`accesskit::ActionRequest`] event. When using an application these would be generated by the
    /// screen reader.
    ActionRequest(accesskit::ActionRequest),
    
    /// A simulated event, e.g. clicks or typing.
```